el-patch.el

```diff
@@ -651,48 +675,55 @@ DEFINITION is a list starting with `defun' or similar."
 (el-patch-deftype defconst
   :classify el-patch-classify-variable
   :locate el-patch-locate-variable
+  :font-lock #'el-patch-fontify-as-variable
```

Comment:
For consistency, I'd like these to be unquoted.

Reply:
Fair enough.  I personally prefer as little magic as possible especially with macros, so I pass quoted or unquoted based on how it is used later.  But consistency almost always trumps purity :)

Reply:
Yes, I think you are right about preferring quoted, and if I were designing this macro again then I would do it your way. It's not a good enough reason to break backwards compatibility, though :P